chore(hybrid-cloud): Add troubleshooting docs for integrations (#5396)
getsentry/sentry-docs: src/docs/product/integrations/troubleshooting.mdx

---
title: Troubleshooting Integrations
sidebar_order: 5000
description: "Learn more about troubleshooting and repairing your integrations"
---

Sentry integrates with many third party services to make your development experience easier. Unfortunately, we can’t always automatically update some of these services when critical data within Sentry is changed. This document is meant to help you navigate any odd behaviour that is to be expected when these settings are changed. In some cases, we may need your team’s manual intervention to get the integration working smoothly again.

> ## If you've changed your organization slug...

Reviewer: ```suggestion
## If you've changed your organization slug
```

> - Any integration that issues links directly to Sentry issues will continue issuing correct links going forward, **but existing links will no longer work as expected**

Reviewer: ```suggestion
- Any integration that create links directly to Sentry issues will continue creating correct links going forward, **but existing links will no longer work as expected**.
```

>   - E.g. Slack notifications containing links to Sentry alerts, Jira tickets which link to Sentry issues

Reviewer: ```suggestion
  - For example, Slack notifications containing links to Sentry alerts, and Jira tickets which link to Sentry issues
```
